Task/Taskfile.yml

```diff
@@ -4,6 +4,107 @@
 version: "3"
 
 tasks:
+  lint:
+    preconditions:
+      - sh: docker pull seiso/goat:latest || echo "Failed to pull the latest goat, continuing anyway..."
+    # Adjustments are passed as vars, not envs, to avoid local workarounds
+    vars:
```

Comment:
This is new; required so we can pass the vars around more directly instead of doing global vars